numpy/ma/tests/test_core.py

```diff
@@ -1990,6 +1991,177 @@ def test_datafriendly_mul_arrays(self):
         assert_equal(a, [[1, 1], [3, 3]])
         assert_equal(a.mask, [[0, 1], [0, 1]])
 
+    def test_inplace_addition_scalar_uint8(self):
```

Comment:
Operators would be tricky, I was just fishing to see if you had any good ideas. Not worth worrying about. Consolidation would be good, just put something in the `err_msg` that let's us know what type failed if they fail.